tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -360,6 +360,36 @@ public function adminDeletesUserUsingTheGraphApi(string $user, ?string $byUser =
 		);
 	}
 
+	/**
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function deleteTheUserUsingTheGraphApi(string $user):void {
+		$this->featureContext->emptyLastHTTPStatusCodesArray();
+		$this->featureContext->emptyLastOCSStatusCodesArray();
+		// Always try to delete the user
+		// users can be deleted using the username in the GraphApi too
+		$this->adminDeletesUserUsingTheGraphApi($user);
+		$this->featureContext->pushToLastStatusCodesArrays();
```

Comment:
helpers should not save any data for later use. do it in the When steps if needed,
```suggestion
		// Always try to delete the user
		// users can be deleted using the username in the GraphApi too
		$this->adminDeletesUserUsingTheGraphApi($user);
```

Reply:
return the response